spec/features/managing_step_by_step_pages_spec.rb

```diff
@@ -309,14 +310,27 @@ def then_I_see_the_step_by_step_page
   end
 
   def then_I_can_see_an_editable_content_section
+    then_I_can_see_a_content_section(editable: true)
+  end
+
+  def then_I_can_see_a_static_content_section
+    then_I_can_see_a_content_section(editable: false)
+  end
+
+  def then_I_can_see_a_content_section(editable:)
     expect(page).to have_css(".gem-c-summary-list")
     within(".gem-c-summary-list") do
       expect(page).to have_content "Content"
-      expect(page).to have_link("Edit", :href => edit_step_by_step_page_path(@step_by_step_page))
       expect(page).to have_content "Title #{@step_by_step_page.title}"
       expect(page).to have_content "Slug #{@step_by_step_page.slug}"
       expect(page).to have_content "Introduction #{@step_by_step_page.introduction}"
       expect(page).to have_content "Description #{@step_by_step_page.description}"
+
+      if editable
```

Comment:
I think for readability this should be split into two separate steps:

`and_I_can_edit_the_metadata` and `and_I_cannot_edit_the_metadata`, with each step being explicit about what can and can't be viewed.

I'm using "metadata" rather than "content" because I think the term content is going to become overloaded and it can also refer to individual and collective step content.